feat(wbh): docs content review (#4428)
scaleway/docs-content: pages/dedibox-network/how-to/subscribe-to-jmrp.mdx

- [Configured the reverse DNS of your IP](/dedibox-ip-failover/how-to/configure-reverse-dns/)
- [Created](https://signup.live.com/signup) a Microsoft account

## Subscribing to JMRP

1. Register your sending IP addresses with [Microsoft's Smart Network Data Services (SNDS)](https://sendersupport.olc.protection.outlook.com/snds/).
2. Navigate to the JMRP section within the SNDS portal.
3. Log in using your Microsoft account credentials.

> 4. Provide the necessary details about your IP addresses that you wish to register with JMRP.

Reviewer: ```suggestion
4. Provide the required details about the IP addresses you wish to register with JMRP.
```
